x-pack/plugins/timelines/public/components/t_grid/toolbar/fields_browser/field_table_header.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useState } from 'react';
+import styled from 'styled-components';
+import {
+  EuiText,
+  EuiPopover,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiContextMenuItem,
+  EuiContextMenuPanel,
+  EuiHorizontalRule,
+} from '@elastic/eui';
+import * as i18n from './translations';
+
+export interface FieldTableHeaderProps {
+  fieldCount: number;
+  filterSelectedEnabled: boolean;
+  setFilterSelectedEnabled: (enabled: boolean) => void;
+}
+
+const Count = styled.span`
+  font-weight: bold;
+`;
+Count.displayName = 'Count';
+
+const FieldTableHeaderComponent: React.FC<FieldTableHeaderProps> = ({
+  fieldCount,
+  filterSelectedEnabled,
+  setFilterSelectedEnabled,
+}) => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  const togglePopover = useCallback(() => {
+    setIsPopoverOpen((open) => !open);
+  }, []);
+
+  const closePopover = useCallback(() => {
+    setIsPopoverOpen(false);
+  }, []);
+
+  return (
+    <EuiFlexGroup>
+      <EuiFlexItem>
+        <EuiText data-test-subj="fields-showing" size="xs">
+          {i18n.FIELDS_SHOWING}
+          <Count data-test-subj="fields-count"> {fieldCount} </Count>
```

Comment:
nit:
Why not `<b>`?
```suggestion
          <b data-test-subj="fields-count"> {fieldCount} </b>
```

Reply:
hmmm I usually try to avoid using styling tags such as `b` or `i`, I prefer to have a semantic markup on one hand and the styling on the other using CSS, and not mix them up, it is more descriptive and intentional this way. 
I would say that this is a best practice, I am not sure honestly. Anyway, I like to keep the different purposes separated 😊